app/src/ui/diff/index.tsx

```diff
@@ -179,9 +179,8 @@ export class Diff extends React.Component<IDiffProps, IDiffState> {
     return (
       <div className="panel empty large-diff">
         <img src={NoDiffImage} className="blankslate-image" alt="" />
+        <p>The diff is too large to be displayed by default.</p>
```

Comment:
Before
![CleanShot 2024-04-15 at 14 26 42](https://github.com/desktop/desktop/assets/75402236/39248e12-2a42-4b5f-ba0c-00241fb25ad0)


After
![Large text diffs with extra space.](https://github.com/desktop/desktop/assets/75402236/840c4096-74aa-48a4-af86-b1fd5ffb7dbf)

From accessibility audit documentation, I have branch https://github.com/githuba11y/repo_the_second2/tree/with-github-desktop-readme.md) for reproducing this screen handy.